docs/admins/howto/new-hub.rst

```diff
@@ -83,9 +112,38 @@ we provide that sets up a blank hub that can be customized.
           name: Deploy <hub-name>
           command: |
             hubploy deploy <hub-name> hub ${CIRCLE_BRANCH}
+		
+		- hubploy/build-image:
```

Comment:
Indentation for this line looks a little off. 'd' from deployment should be under the 'b' in hubploy.

Reply:
Done